src/util.h

```diff
@@ -11,10 +11,16 @@
 #include "libsecp256k1-config.h"
 #endif
 
+#include <string.h>
 #include <stdlib.h>
 #include <stdint.h>
 #include <stdio.h>
 
+#if defined(_MSC_VER)
+// For SecureZeroMemory
+#include <Windows.h>
```

Comment:
I think that's okay but I'm not entirely sure if we want that. It's used in Bitcoin Core too.

Reply:
I would be that this should be version gated, I'll see if I can figure out where thats supported to.